bokeh/document.py

```diff
@@ -752,10 +752,6 @@ def validate(self):
             None
 
         '''
-        # logging.basicConfig is a no-op if there's already
-        # some logging configured. We want to make sure warnings
-        # go somewhere so configure here if nobody has.
-        logging.basicConfig(level=logging.INFO)
```

Comment:
`validate` is not some random function. It is called at serialization time. There is basically no usage of Bokeh that will not go through this function. I'm open to suggestions. I don't care about calling or not calling `basicConfig` especially, all that matters is that these usage validation message are not silent by default under any circumstance.